selfdrive/controls/lib/lateral_planner.py

```diff
@@ -42,8 +44,12 @@ def update(self, sm):
     self.LP.parse_model(md)
     if len(md.position.x) == TRAJECTORY_SIZE and len(md.orientation.x) == TRAJECTORY_SIZE:
       self.path_xyz = np.column_stack([md.position.x, md.position.y, md.position.z])
+      self.speed_forward = np.linalg.norm(np.column_stack([md.velocity.x, md.velocity.y, md.velocity.z]), axis=1)
       self.t_idxs = np.array(md.position.t)
       self.plan_yaw = list(md.orientation.z)
+      self.plan_yaw_rate = list(md.orientationRate.z)
+      self.lateral_acc = self.speed_forward * self.plan_yaw_rate
+      self.jerk = np.gradient(self.lateral_acc, self.t_idxs)
```

Comment:
should give a better approximation at t0. 
center points are already evaluated with second order approximation but default for the edges is 1st. 

```suggestion
      self.jerk = np.gradient(self.lateral_acc, self.t_idxs, edge_order=2)
```